File: iosApp/iosApp/feature/profile/order/order_list/OrderItem.swift
//
//  OrderItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 12.03.2022.
//

import shared
import SwiftUI

struct OrderItem: Identifiable {
    let id: String
    let status: OrderStatus
    let code: String
    let dateTime: String
}

File: shared/src/commonMain/kotlin/com/bunbeauty/shared/ui/screen/order/model/OrderItem.kt
package com.bunbeauty.shared.ui.screen.order.model

import com.bunbeauty.core.model.order.OrderStatus

data class OrderItem(
    val uuid: String,
    val status: OrderStatus,
    val statusName: String,
    val code: String,
    val dateTime: String,
)
